### PR TITLE
Task/flutter gen

### PR DESCRIPTION
**Pull request by ikennaumeh**
**This PR was created to**

- include flutter_gen
- replace strings in start up view with generated assets.

### CODE REVIEW

File: .fvm/flutter_sdk [hunk 1]
```diff
@@ -0,0 +1 @@
+C:/Users/Admin/fvm/versions/3.3.10
```
Comment (isaacadariku) [inline .fvm/flutter_sdk]: This is not supposed to be committed. It is already ignored 🤔 Please remove it from the history.

File: .fvm/fvm_config.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "flutterSdkVersion": "3.3.10",
+  "flavors": {}
+}
```
Comment (isaacadariku) [inline .fvm/fvm_config.json]: This file too

File: .gitignore [hunk 3]
```diff
@@ -8,6 +8,8 @@ build/
 .idea
 # If you're building an application, you may want to check-in your pubspec.lock
 pubspec.lock
+macos/Flutter/
+windows/flutter/
```
Comment (isaacadariku) [inline .gitignore]: We don't have to do this, because we may need some other time. But is your branch not stable with dev

File: .idea/.gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (isaacadariku) [inline .idea/.gitignore:3]: Why ignore this files?

**Review by isaacadariku — CHANGES_REQUESTED** [review]
It looks good, but so many unnecessary files, I am not sure why but please remove them and let the PR be in it's scope.

File: .gitignore (hunk 3 above)
Comment (ikennaumeh) [reply .gitignore]: Okay, I think i'd stop doing "git add ." that adds up all the files

File: .idea/.gitignore (hunk 4 above)
Comment (ikennaumeh) [reply .idea/.gitignore:3]: No reason. I can remove ".idea" from gitignore if you want.

**Pull request closed without merging**